keyboards/vortex/m0110a/m0110a_ansi/info.json

```diff
@@ -0,0 +1,135 @@
+                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 3 },
+                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 3 },
+                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 3 },
+                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 3 },
+                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 3 },
+                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 3 },
+                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 3 },
+                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 3 },
+                { "label": "Enter", "matrix": [10, 4], "w": 2.25, "x": 12.75, "y": 3 },
+                { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 3 },
+                { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 3 },
+                { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 3 },
+
+                { "label": "Shift", "matrix": [0, 0], "w": 2.25, "x": 0, "y": 4 },
+                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 4 },
+                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 4 },
+                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 4 },
+                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 4 },
+                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 4 },
+                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 4 },
+                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 4 },
+                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 4 },
+                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 4 },
+                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 4 },
+                { "label": "Shift", "matrix": [9, 1], "w": 2.75, "x": 12.25, "y": 4 },
+                { "label": "Num1", "matrix": [11, 2], "x": 15.5, "y": 4 },
+                { "label": "Num2", "matrix": [12, 2], "x": 16.5, "y": 4 },
+                { "label": "Num3", "matrix": [13, 2], "x": 17.5, "y": 4 },
+                { "label": "Numenter", "matrix": [13, 6], "h":2, "x": 18.5, "y": 4 },
+
+                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 5 },
+                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 5 },
+                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 5 },
+                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 5 },
+                { "label": "RAlt", "matrix": [9, 5], "w": 1.25, "x": 10, "y": 5 },
+                { "label": "RWin", "matrix": [8, 4], "w": 1.25, "x": 11.25, "y": 5 },
+                { "label": "RCtrl", "matrix": [0, 4], "w": 1.25, "x": 12.5, "y": 5 },
+                { "label": "FN", "matrix": [9, 2], "w": 1.25, "x": 13.75, "y": 5 },
+                { "label": "Num0", "matrix": [12, 3], "w":2, "x": 15.5, "y": 5 },
+                { "label": "Numdel", "matrix": [13, 3], "x": 17.5, "y": 5 }
```

Comment:
1. added layouts from PRs #21932, #21933, and #21934
2. added layout `LAYOUT_all` to conform to PR guidelines
3. for ISO layouts, moved ISO Enter key position in matrix/layout sequence to conform to community layouts
```suggestion
        "LAYOUT_all": {
            "layout": [
                { "label": "ESC", "matrix": [1, 6], "x": 0, "y": 0 },
                { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
                { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
                { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
                { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
                { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
                { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
                { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
                { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
                { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
                { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
                { "label": "\\", "matrix": [10, 2], "x": 13, "y": 0 },
                { "label": "delete", "matrix": [6, 5], "x": 14, "y": 0 },
                { "label": "Numlock", "matrix": [11, 4], "x": 15.5, "y": 0 },
                { "label": "Num/", "matrix": [12, 4], "x": 16.5, "y": 0 },
                { "label": "Num*", "matrix": [13, 4], "x": 17.5, "y": 0 },
                { "label": "Num-", "matrix": [13, 5], "x": 18.5, "y": 0 },

                { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
                { "label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "Num7", "matrix": [11, 0], "x": 15.5, "y": 1 },
                { "label": "Num8", "matrix": [12, 0], "x": 16.5, "y": 1 },
                { "label": "Num9", "matrix": [13, 0], "x": 17.5, "y": 1 },
                { "label": "Num+", "matrix": [11, 3], "h":2, "x": 18.5, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
                { "label": "Enter", "matrix": [10, 4], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
                { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
                { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },

                { "label": "Shift", "matrix": [0, 0], "w": 1.25, "x": 0, "y": 3 },
                { "label": "\\", "matrix": [2, 3], "x": 1.25, "y": 3 },
                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 3 },
                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 3 },
                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 3 },
                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [9, 1], "w": 1.75, "x": 12.25, "y": 3 },
                { "label": "Up", "matrix": [3, 5], "x": 14.25, "y": 3.25 },
                { "label": "Num1", "matrix": [11, 2], "x": 15.5, "y": 3 },
                { "label": "Num2", "matrix": [12, 2], "x": 16.5, "y": 3 },
                { "label": "Num3", "matrix": [13, 2], "x": 17.5, "y": 3 },
                { "label": "Numenter", "matrix": [13, 6], "h":2, "x": 18.5, "y": 3 },

                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "RAlt", "matrix": [9, 5], "x": 10, "y": 4 },
                { "label": "FN", "matrix": [9, 2], "x": 11, "y": 4 },
                { "label": "RCtrl", "matrix": [0, 4], "x": 12, "y": 4 },
                { "label": "Left", "matrix": [0, 3], "x": 13.25, "y": 4.25 },
                { "label": "Down", "matrix": [7, 3], "x": 14.25, "y": 4.25 },
                { "label": "Right", "matrix": [0, 5], "x": 15.25, "y": 4.25 },
                { "label": "Num0", "matrix": [12, 3], "x": 16.5, "y": 4 },
                { "label": "Numdel", "matrix": [13, 3], "x": 17.5, "y": 4 }
            ]
        },
        "LAYOUT_ansi": {
            "layout": [
                { "label": "ESC", "matrix": [1, 6], "x": 0, "y": 0 },
                { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
                { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
                { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
                { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
                { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
                { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
                { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
                { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
                { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
                { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [10, 1], "w":2, "x": 13, "y": 0 },
                { "label": "Numlock", "matrix": [11, 4], "x": 15.5, "y": 0 },
                { "label": "Num/", "matrix": [12, 4], "x": 16.5, "y": 0 },
                { "label": "Num*", "matrix": [13, 4], "x": 17.5, "y": 0 },
                { "label": "Num-", "matrix": [13, 5], "x": 18.5, "y": 0 },

                { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
                { "label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "Num7", "matrix": [11, 0], "x": 15.5, "y": 1 },
                { "label": "Num8", "matrix": [12, 0], "x": 16.5, "y": 1 },
                { "label": "Num9", "matrix": [13, 0], "x": 17.5, "y": 1 },
                { "label": "Num+", "matrix": [11, 3], "h":2, "x": 18.5, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
                { "label": "Enter", "matrix": [10, 4], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
                { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
                { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },

                { "label": "Shift", "matrix": [0, 0], "w": 2.25, "x": 0, "y": 3 },
                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 3 },
                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 3 },
                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 3 },
                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [9, 1], "w": 2.75, "x": 12.25, "y": 3 },
                { "label": "Num1", "matrix": [11, 2], "x": 15.5, "y": 3 },
                { "label": "Num2", "matrix": [12, 2], "x": 16.5, "y": 3 },
                { "label": "Num3", "matrix": [13, 2], "x": 17.5, "y": 3 },
                { "label": "Numenter", "matrix": [13, 6], "h":2, "x": 18.5, "y": 3 },

                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "RAlt", "matrix": [9, 5], "w": 1.25, "x": 10, "y": 4 },
                { "label": "RWin", "matrix": [8, 4], "w": 1.25, "x": 11.25, "y": 4 },
                { "label": "RCtrl", "matrix": [0, 4], "w": 1.25, "x": 12.5, "y": 4 },
                { "label": "FN", "matrix": [9, 2], "w": 1.25, "x": 13.75, "y": 4 },
                { "label": "Num0", "matrix": [12, 3], "w":2, "x": 15.5, "y": 4 },
                { "label": "Numdel", "matrix": [13, 3], "x": 17.5, "y": 4 }
            ]
        },
        "LAYOUT_iso": {
            "layout": [
                { "label": "ESC", "matrix": [1, 6], "x": 0, "y": 0 },
                { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
                { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
                { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
                { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
                { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
                { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
                { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
                { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
                { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
                { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [10, 1], "w":2, "x": 13, "y": 0 },
                { "label": "Numlock", "matrix": [11, 4], "x": 15.5, "y": 0 },
                { "label": "Num/", "matrix": [12, 4], "x": 16.5, "y": 0 },
                { "label": "Num*", "matrix": [13, 4], "x": 17.5, "y": 0 },
                { "label": "Num-", "matrix": [13, 5], "x": 18.5, "y": 0 },

                { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
                { "label": "Num7", "matrix": [11, 0], "x": 15.5, "y": 1 },
                { "label": "Num8", "matrix": [12, 0], "x": 16.5, "y": 1 },
                { "label": "Num9", "matrix": [13, 0], "x": 17.5, "y": 1 },
                { "label": "Num+", "matrix": [11, 3], "h":2, "x": 18.5, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
                { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 2 },
                { "label": "Enter", "matrix": [10, 4], "w": 1.25, "h": 2, "x": 13.75, "y": 1 },
                { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
                { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
                { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },

                { "label": "Shift", "matrix": [0, 0], "w": 1.25, "x": 0, "y": 3 },
                { "label": "\\", "matrix": [2, 3], "x": 1.25, "y": 3 },
                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 3 },
                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 3 },
                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 3 },
                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [9, 1], "w": 2.75, "x": 12.25, "y": 3 },
                { "label": "Num1", "matrix": [11, 2], "x": 15.5, "y": 3 },
                { "label": "Num2", "matrix": [12, 2], "x": 16.5, "y": 3 },
                { "label": "Num3", "matrix": [13, 2], "x": 17.5, "y": 3 },
                { "label": "Numenter", "matrix": [13, 6], "h":2, "x": 18.5, "y": 3 },

                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "RAlt", "matrix": [9, 5], "w": 1.25, "x": 10, "y": 4 },
                { "label": "RWin", "matrix": [8, 4], "w": 1.25, "x": 11.25, "y": 4 },
                { "label": "RCtrl", "matrix": [0, 4], "w": 1.25, "x": 12.5, "y": 4 },
                { "label": "FN", "matrix": [9, 2], "w": 1.25, "x": 13.75, "y": 4 },
                { "label": "Num0", "matrix": [12, 3], "w":2, "x": 15.5, "y": 4 },
                { "label": "Numdel", "matrix": [13, 3], "x": 17.5, "y": 4 }
            ]
        },
        "LAYOUT_1800_ansi": {
            "layout": [
                { "label": "ESC", "matrix": [1, 6], "x": 0, "y": 0 },
                { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
                { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
                { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
                { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
                { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
                { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
                { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
                { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
                { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
                { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [10, 1], "w":2, "x": 13, "y": 0 },
                { "label": "Numlock", "matrix": [11, 4], "x": 15.5, "y": 0 },
                { "label": "Num/", "matrix": [12, 4], "x": 16.5, "y": 0 },
                { "label": "Num*", "matrix": [13, 4], "x": 17.5, "y": 0 },
                { "label": "Num-", "matrix": [13, 5], "x": 18.5, "y": 0 },

                { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
                { "label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "Num7", "matrix": [11, 0], "x": 15.5, "y": 1 },
                { "label": "Num8", "matrix": [12, 0], "x": 16.5, "y": 1 },
                { "label": "Num9", "matrix": [13, 0], "x": 17.5, "y": 1 },
                { "label": "Num+", "matrix": [11, 3], "h":2, "x": 18.5, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
                { "label": "Enter", "matrix": [10, 4], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
                { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
                { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },

                { "label": "Shift", "matrix": [0, 0], "w": 2.25, "x": 0, "y": 3 },
                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 3 },
                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 3 },
                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 3 },
                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [9, 1], "w": 1.75, "x": 12.25, "y": 3 },
                { "label": "Up", "matrix": [3, 5], "x": 14.25, "y": 3.25 },
                { "label": "Num1", "matrix": [11, 2], "x": 15.5, "y": 3 },
                { "label": "Num2", "matrix": [12, 2], "x": 16.5, "y": 3 },
                { "label": "Num3", "matrix": [13, 2], "x": 17.5, "y": 3 },
                { "label": "Numenter", "matrix": [13, 6], "h":2, "x": 18.5, "y": 3 },

                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "RAlt", "matrix": [9, 5], "x": 10, "y": 4 },
                { "label": "FN", "matrix": [9, 2], "x": 11, "y": 4 },
                { "label": "RCtrl", "matrix": [0, 4], "x": 12, "y": 4 },
                { "label": "Left", "matrix": [0, 3], "x": 13.25, "y": 4.25 },
                { "label": "Down", "matrix": [7, 3], "x": 14.25, "y": 4.25 },
                { "label": "Right", "matrix": [0, 5], "x": 15.25, "y": 4.25 },
                { "label": "Num0", "matrix": [12, 3], "x": 16.5, "y": 4 },
                { "label": "Numdel", "matrix": [13, 3], "x": 17.5, "y": 4 }
            ]
        },
        "LAYOUT_1800_iso": {
            "layout": [
                { "label": "ESC", "matrix": [1, 6], "x": 0, "y": 0 },
                { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
                { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
                { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
                { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
                { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
                { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
                { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
                { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
                { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
                { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [10, 1], "w":2, "x": 13, "y": 0 },
                { "label": "Numlock", "matrix": [11, 4], "x": 15.5, "y": 0 },
                { "label": "Num/", "matrix": [12, 4], "x": 16.5, "y": 0 },
                { "label": "Num*", "matrix": [13, 4], "x": 17.5, "y": 0 },
                { "label": "Num-", "matrix": [13, 5], "x": 18.5, "y": 0 },

                { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
                { "label": "Num7", "matrix": [11, 0], "x": 15.5, "y": 1 },
                { "label": "Num8", "matrix": [12, 0], "x": 16.5, "y": 1 },
                { "label": "Num9", "matrix": [13, 0], "x": 17.5, "y": 1 },
                { "label": "Num+", "matrix": [11, 3], "h":2, "x": 18.5, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
                { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 2 },
                { "label": "Enter", "matrix": [10, 4], "w": 1.25, "h": 2, "x": 13.75, "y": 1 },
                { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
                { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
                { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },

                { "label": "Shift", "matrix": [0, 0], "w": 1.25, "x": 0, "y": 3 },
                { "label": "\\", "matrix": [2, 3], "x": 1.25, "y": 3 },
                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 3 },
                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 3 },
                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 3 },
                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [9, 1], "w": 1.75, "x": 12.25, "y": 3 },
                { "label": "Up", "matrix": [3, 5], "x": 14.25, "y": 3.25 },
                { "label": "Num1", "matrix": [11, 2], "x": 15.5, "y": 3 },
                { "label": "Num2", "matrix": [12, 2], "x": 16.5, "y": 3 },
                { "label": "Num3", "matrix": [13, 2], "x": 17.5, "y": 3 },
                { "label": "Numenter", "matrix": [13, 6], "h":2, "x": 18.5, "y": 3 },

                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "RAlt", "matrix": [9, 5], "x": 10, "y": 4 },
                { "label": "FN", "matrix": [9, 2], "x": 11, "y": 4 },
                { "label": "RCtrl", "matrix": [0, 4], "x": 12, "y": 4 },
                { "label": "Left", "matrix": [0, 3], "x": 13.25, "y": 4.25 },
                { "label": "Down", "matrix": [7, 3], "x": 14.25, "y": 4.25 },
                { "label": "Right", "matrix": [0, 5], "x": 15.25, "y": 4.25 },
                { "label": "Num0", "matrix": [12, 3], "x": 16.5, "y": 4 },
                { "label": "Numdel", "matrix": [13, 3], "x": 17.5, "y": 4 }
```